dls-controls/malcolmjs
# Feature/hidden link info #323

## Description

Improved hidden links to allow them to be selected, display info and be deleted as per normal links.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/gui/PANDA/layout

## Agile board tracking

connect to #323 
closes #323 
connect to #396 
closes #396
